tests/unit/UnitTests.cmake

```diff
@@ -18,6 +18,7 @@ target_compile_definitions(test_lazysmallbitvector
 target_include_directories(test_lazysmallbitvector
   PRIVATE "${CMAKE_SOURCE_DIR}")
 target_link_libraries(test_lazysmallbitvector
+  revngModel
```

Comment:
I've another commit to fix all of this, I'll import that one instead.

Reply:
Ok. But first give me a reference to it please, and let me test it. I've been bugged by broken linking for a while now, especially when linking `revng`'s stuff from external projects, and I really really want this to be fixed this for good.

Reply:
I'm testing with `-Wl,--no-undefined`. It should be gone for good.